config/locales/en.yml

```
@@ -32,6 +32,7 @@
 en:
   hello: "Hello world"
   csv:
+    upload_success: 'Your file has been uploaded!'
```

```suggestion
    upload_success: 'Your file has been uploaded successfuly!'
```

Optional suggestion :)